### PR TITLE
executor: support table reader executor read from tiflash.

### PR DESCRIPTION
**Pull request by lzmhhh123**
### What problem does this PR solve? 
Support table reader executor read from tiflash.

### What is changed and how it works?
Set store type for the `Request` and move `tikv.StoreType` to `kv.StoreType` to avoid import circle.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has persistent data change

Side effects

 - Increased code complexity

### CODE REVIEW

**Comment by zz-jason** [comment]
@lzmhhh123 CI failed.

**Comment by lzmhhh123** [comment]
@zz-jason ci has been fixed.

File: store/tikv/coprocessor.go [hunk 1]
```diff
@@ -211,7 +211,7 @@ func (r *copRanges) split(key []byte) (*copRanges, *copRanges) {
 // rangesPerTask limits the length of the ranges slice sent in one copTask.
 const rangesPerTask = 25000
 
-func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, desc bool, streaming bool) ([]*copTask, error) {
+func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, desc bool, streaming bool, storeType kv.StoreType) ([]*copTask, error) {
```
Comment (zz-jason) [inline store/tikv/coprocessor.go]: there are 6 parameters on this function, can we shrink the last three parameter to `req *kv.Request`?

**Pull request merged**